.github/workflows/Automation Master Legacy Workflow.yml

```diff
@@ -125,6 +126,41 @@ jobs:
         if: inputs.owner == 'SkylineCommunications'
         run: nuget sources add -src "https://nuget.pkg.github.com/SkylineCommunications/index.json" -UserName USERNAME -Password ${{ secrets.GITHUB_TOKEN }}
 
+
+      - name: Enable Skyline Azure Cloud NuGet Registry
+        env:
+          AZURE_TOKEN_EXISTS: ${{ secrets.azureToken }}
+        if: env.AZURE_TOKEN_EXISTS != null && inputs.owner == 'SkylineCommunications'
```

Comment:
Why the environment stuff here? Cause you should be able to use the ${{ secrets.azureToken }} directly in the if